contracts/modules/TransferManager/SingleTradeVolumeRestrictionManager.sol

```diff
@@ -0,0 +1,176 @@
+pragma solidity ^0.4.24;
+import "./ITransferManager.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+/**
+ * @title Transfer Manager for limiting volume of tokens in a single trade
+ */
+
+contract SingleTradeVolumeRestrictionManager is ITransferManager {
+  using SafeMath for uint256;
+
+  bool public isTransferLimitInPercentage;
+  uint256 public globalTransferLimit;
+
+  // should be multipled by 10^16. if the transfer percentage is 20%, then globalTransferLimitInPercentage should be 20*10^16
+  uint256 public globalTransferLimitInPercentage;
+
+  mapping(address=>bool) public exemptWallets;
+
+  mapping(address => bool) public specialTransferLimitWallets;
+
+  mapping(address => uint) public specialTransferLimits;
+
+  event LogExemptWalletAdded(address _wallet);
```

Comment:
Since some time ago solidity introduced the "emit" keyword, we recently changed all events so they don't start with "log", please remove "Log" prefix from every event.